## üö® Attention: pre-course action

In this section, we will perform protein structure predictions using Colab multimer (for AF2), AF3 server, and Tamarinde Bio (for Boltz-2 and Chai-1). Please create accounts for the servers in advance of the session through the links provided on the README page. We will commence the predictions during the workshop session of the course.

##  üìù Take-home work (20 pts)

This workshop presents a case study in which we will perform structure predictions over an antibody‚Äìantigen (Ab‚ÄìAg) complexe that was entirely blind by all these prediction methods, in other words, the complex was released after the training cut-off of these methods.

- Our candidate, PDB ID 7ZK1, is the crystal structure of cystinosin, a proton-driven cystine transporter, bound to both a sybody and a nanobody. Here what it looks like:

  <img width="250" height="371" alt="image1-2" src="https://github.com/user-attachments/assets/b822be23-8672-4e5e-89bb-221ef7bfc4f4" />

- For simplicity, let's assume that we are only interested in interaction between the synthetic nanobody (sybody) and the transporter.

1) Please retrieve the structure from the PDB along with the FASTA sequences for all chains.

2) Perform complex prediction with AF2 via Colab (1 pts)
   - Employ multimer_v3 model 
   - Adjust # of recycles to 6. (Attention: Only one prediction can be run at a time in Colab.)

3) Perform complex predictions with AF3 via AF3 server (1 pts)
   - Adjust # of seeds to 1 and 20, respectively. (2 different predicitons will be obtained)

4) Perform complex predictions with Chai-1 via Tamarind Bio (1 pts)
   - Adjust # of seeds to 1 and # of recycles to 20
   - Apply a pocket restraint to the antibody
   - Please assign unique job names in Tamarind Bio, as jobs with identical names will be overwritten.

5) Perform complex predictions with Boltz-2 via Tamarind Bio (1 pts)
   - Adjust # of seeds to 1 and # of recycles to 20
   - Apply a contacts restraint to the antibody
   - Please assign unique job names in Tamarind Bio, as jobs with identical names will be overwritten.


6) Once the predictions have been compiled, evaluate the different ranks within each prediction and extract the highest-ranked models. For AlphaFold2 predictions generated in Colab, a ranking will already be provided for the five distinct model outputs. For models generated by other methods, select up to three representative models from the major clusters. If no major structural differences are observed among the models produced by a method, select a single representative model for further analysis.

7) Evaluate the accuracy of selected models (at least one per method) based on the pLDDT, PAE, and pTM scores provided in the corresponding .json files. (4 pts, 1 pts for each method)

8) Calculate the RMSD of the generated models by aligning them to the ground truth structure (7zk1). Perform the RMSD calculations in two distinct alignment modes: (i) aligned on the antigen and (ii)aligned on the antibody. Report the global RMSD values and interpret them in the context of other structural accuracy metrics for comparative evaluation. (ChimeraX's matchmaker module is an easy option to calculate RMSDs) (4 pts, 1 pts for each method)

9) Using the IMGT/DomainGapAlign tool, identify the CDR regions (CDR1, CDR2, and CDR3) based on the sequence. (1 pts, 0.25 pts for each method)
Link to the IMGT/DomainGapAlign tool: https://www.imgt.org/3Dstructure-DB/cgi/DomainGapAlign.cgi

10) Calculate RMSDs separately for the identified CDR regions (CDR1, CDR2, and CDR3) when the complex is aligned on the antibody. (1 pts, 0.25 pts for each method)

11) Report the membrane localization of the transporter using TmAlphaFold. (1 pts)

12) Provide a brief discussion on the accuracy of the predicted antibody and antigen structures, the overall accuracy of the predicted complex, and the accuracy of the CDR loops, highlighting their contributions to protein‚Äìprotein docking. (5 pts)

üçÄ Bonus Question: While AF3 and AF3-implementations represent the latest advancements in structure prediction, building upon the foundations of AF2, can one assume a straightforward positive correlation between the performance of these methods and their chronological development from earlier to more recent models? Discuss briefly, in a maximum of five sentences, and provide a reference paper to support your arguments. (+ 2 pts)

## üè† Take-home points

- Adding lipids can help support the correct orientation of membrane-embedded proteins.

- Adding salts can help proteins fold into the native conformation.

- Natural ligands can provide structural information that guides the positioning of specific ligand backbones.

- Be aware of hallucinations when predicting structures of larger protein sequences.

- Multiple copies may superimpose during the prediction of higher-order oligomeric assemblies.

- 1/20 seeds got confident prediction scores.

## Summary

This section focuses on structure predictions using four different methods: AF2, AF3, Boltz-2,
and Chai-1. The goal is to evaluate how various parameters such as model type, number of recycles, number of seeds, presence of ions, presence of lipids, and the introduction of pocket and contact restraints affect the prediction outcomes. We will also discuss practical hacks and warnings to consider during protein structure prediction.

<img width="672" height="367" alt="Screenshot 2025-08-06 at 08 25 01" src="https://github.com/user-attachments/assets/90fc4ece-9bf8-48fc-9658-6fd830207281" />
 from Sergey Ovchinnikov‚Äôs slides

## References

Sergey Ovchinnikov‚Äôs slides

## Slides

https://docs.google.com/presentation/d/1MeapGNPZwZcQyvFA1ts-XJdXpp9gv7b2nevZYu8CEAc/edit?usp=sharing
