## üìù Workshop report (The deadline is 12:00 a.m. on the same day) (10 pts)

In this workshop session, we will perform protein‚Äìprotein docking experiments of IL37 (monomeric and dimeric form) and its primary receptor IL18RŒ± using four different approaches: rigid-body docking, flexible docking, ensemble docking, and ML‚Äìbased models. This complex is known to block inflammation mediated by IL18 by binding to its receptor and preventing the formation of IL18 receptor complexes. 

- Please retrieve the crystal structure with PDB ID 3wo3 as the reference complex of the IL18 ligand bound to the IL18RŒ± receptor and the crystal structure with PDB ID 5hn1 as the reference for the IL37. 

- Using ClusPro, perform docking of IL37 in its monomeric form to the IL18RŒ± receptor. Briefly explain the results and discuss the biological relevance of the docked-complex by considering the structural homology between IL18 and IL37. (1 pts)

- Using HADDOCK, perform docking of IL37 in its monomeric form to the IL18RŒ± receptor. Briefly explain the results and discuss the biological relevance of the docked-complex by considering the structural homology between IL18 and IL37. (1 pts)

- Using the provided receptor conformations extracted from MD simulations, perform docking of dimeric IL37 to the IL18RŒ± receptor via the HADDOCK server. Briefly explain the results and discuss the biological relevance of the docked complex, highlighting the role of different receptor conformations in the docking process. (2.5 pts)

- Obtain sequences for IL37 and IL18RŒ± from the UniProt (please pay attention to the sequence ranges in crystal structures), predict the IL37‚ÄìIL18RŒ± complex with Boltz-2 on TamarindBio and calculate the binding affinity. Briefly explain the results and discuss the biological relevance of the predicted complex. (1.5 pts)

- In FoldX, use the AnalyseComplex command to calculate the interaction energies of complexes generated by the different docking methods. Discuss the interaction energies and biological relevance of the complexes, considering the structural homology between IL18 and IL37. Which method showed the best fit to our hypothesis? What might be the reason behind the superior performance of this method compared to others?  (4 pts)
  

## üè† Take-home points

- Protein-protein interaction (PPI) prediction problem is the challenge of predicting whether and how two proteins interact, often from sequence or structural information, given the complexity and variability of biological interfaces.

- Rigid-body docking assumes proteins remain structurally rigid and searches for the best geometric and physicochemical fit between their surfaces.

- Flexible docking allows conformational changes, typically in side chains or loops, to better capture realistic protein‚Äìprotein interactions.

- Ensemble docking uses multiple conformations of proteins (from experiments or simulations) to account for structural flexibility during docking.

- Machine learning approaches learn interaction patterns directly from large datasets of protein sequences and structures, enabling end-to-end prediction of binding likelihoods and interfaces.

## Summary

In this section, we will discuss various protein‚Äìprotein docking methods, their theoretical foundations, and applications. Protein‚Äìprotein docking is a computational strategy used to predict the structures of protein complexes from known subunits and represents an upper-level challenge built on protein structure prediction. The field has evolved over several decades, beginning with rigid-body docking in the 1970s‚Äì1980s, which relied on surface complementarity, advancing to scoring-based and flexible docking in the 1990s‚Äì2000s to account for side-chain and backbone motions, and later to ensemble docking, which incorporates conformational variability. Most recently, deep learning‚Äìbased methods such as AlphaFold-Multimer have achieved unprecedented accuracy by leveraging large-scale sequence and structure datasets.

![ProteinComplexStructure-2](https://github.com/user-attachments/assets/c42dcaed-9135-4031-be60-9a4edab16670)

from Stony Brook University

## Softwares

Link to the ClusPro server: https://cluspro.bu.edu/home.php

Link to the HADDOCK web server: https://rascar.science.uu.nl/haddock2.4/submit/1

Link to the Tamarind Bio server: https://www.tamarind.bio/

## References

Link to the paper Machine learning to predict de novo protein‚Äìprotein interactions: https://www.cell.com/trends/biotechnology/fulltext/S0167-7799%2825%2900158-1

Link to the paper Modulation of p53 binding to MDM2: computational studies reveal important roles of Tyr100: https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-10-S15-S6#Sec9

Link to the paper Calcium Signalling in Heart and Vessels: Role of Calmodulin and Downstream Calmodulin-Dependent Protein Kinases: https://www.mdpi.com/1422-0067/23/24/16139

Link to the paper of Elucidation of protein function using computational docking and hotspot analysis by ClusPro and FTMap: https://journals.iucr.org/d/issues/2022/06/00/qg5007/qg5007fig1.html

Link to the active and passive residue selection in HADDOCK: https://www.bonvinlab.org/haddock-restraints/active_passive.html

Link to the paper on Modeling Protein‚ÄìGlycan Interactions with HADDOCK: https://pubs.acs.org/doi/10.1021/acs.jcim.4c01372
