## 🏠 Take-home Points

- AlphaFold provides several confidence metrics such as pLDDT, ipLDDT, PAE, iPAE, pTM, and ipTM to help assess the quality and reliability of its accuracy of protein structures.

- The pLDDT score provides local (per-residue) confidence, reflecting the accuracy of predicted secondary structures, while ipLDDT offers an analogous measure for interface regions.

          - pLDDT>90: very high
          - 70<pLDDT<90: confident
          - 50<pLDDT<70: low
          - pLDDT<50: very low
  

- The PAE (Predicted Aligned Error) score, a continuous metric, reflects the confidence in the relative positioning between pairs of residues (with iPAE used for interface regions). It is particularly informative for domain–domain positioning, multimer interface orientation, flexible or hinge regions.

- The pTM (predicted TM-score) provides an estimate of the global structural accuracy of a predicted model, evaluating the correctness of the overall protein fold .
  
          - pTM>0.5: similar fold
          - pTM<0.5: wrong fold



## Summary
Deep learning methods such as AlphaFold and RoseTTAFold have dramatically advanced ability to predict the structures of proteins and protein–protein complexes. However, accurately evaluating the confidence and accuracy of these predictions is equally important, especially for guiding experimental validation or downstream applications like drug design and variant analysis. Confidence metrics such as pLDDT and PAE, generated by the models themselves, provide residue-level and inter-residue/inter-domain error estimates. For protein complexes, additional scores like the predicted TM-score (pTM), interface predicted TM-score (ipTM) help assess global and interface-specific accuracy. Therefore, thorough assessment of both global and local predicted structural motifs is critical for accurately interpreting and validating deep learning–based structural predictions.

![sualization-of-Predicted-Aligned-Errors-Protein-specific-pages-contain-an-interactive-2D](https://github.com/user-attachments/assets/2cbdb114-0ed0-45e8-ad5e-2311d0bf38d5)

## References

Link to the paper on lDDT metric: https://academic.oup.com/bioinformatics/article/29/21/2722/195896?login=true

Link to thr paper of ColabFold: https://www.nature.com/articles/s41592-022-01488-1

pLDDT score:

   ➡️ Link to the AF2 structure AF-Q8IZS7-F1: https://alphafold.ebi.ac.uk/entry/Q8IZS7

   ➡️ Link to the AF2 structure AF-Q3S2X4-F1: https://alphafold.ebi.ac.uk/entry/Q3S2X4

   ➡️ Link to the AF2 structure AF-P36883-F1: https://alphafold.ebi.ac.uk/entry/P36883

PAE score:

   ➡️ Link to the AF2 structure AF-Q9NRI5-F1: https://alphafold.ebi.ac.uk/entry/Q9NRI5

   ➡️ Link to the AF2 structure AF-P9WNQ7-F1: https://alphafold.ebi.ac.uk/entry/P9WNQ7

   ➡️ Link to the AF2 structure AF-O15121-F1: https://alphafold.ebi.ac.uk/entry/O15121

Link to the paper "How significant is a protein structure similarity with TM-score = 0.5?" : https://academic.oup.com/bioinformatics/article/26/7/889/213219
   
Link to the paper "Scoring function for automated assessment of protein structure template quality": https://onlinelibrary.wiley.com/doi/full/10.1002/prot.20264?casa_token=hX_5at8e3FsAAAAA%3AKUNF1hwKQIVYIOGv2noTxnulTMCVb5ut5vLXkZ-eIuajcLKghchnCjsD97mN91Qw2Y6HQocFjxEU0x5P

Link to the paper "Interrogation and validation of the interactome of neuronal Munc18-interacting Mint proteins with AlphaFold2": https://www.biorxiv.org/lookup/doi/10.1101/2023.02.20.529329
